chamber/models/fields.py

```diff
@@ -194,3 +196,44 @@ class EnumSequencePositiveIntegerField(EnumSequenceFieldMixin, models.PositiveIn
 
 class EnumSequenceCharField(EnumSequenceFieldMixin, models.CharField):
     pass
+
+
+class PriceField(DecimalField):
+
+    def __init__(self, *args, **kwargs):
+        self.currency = kwargs.pop('currency', ugettext('CZK'))
+        custom_kwargs = kwargs.copy()
+        custom_kwargs.update({
+            'decimal_places': 2,
+            'max_digits': 10,
```

Comment:
Chápu. Já jsem chtěl zachovat stejnou funkcionalitu, jako to teď máme (tzn. teď je to v KNJ teda špatně).